src/subdomains/generic/kyc/services/name-check.service.ts

```diff
@@ -101,6 +111,23 @@ export class NameCheckService implements OnModuleInit {
     return RiskStatus.SANCTIONED;
   }
 
+  async getRiskDataAndUploadPdf(userData: UserData, name: string, dob?: Date): Promise<DilisenseApiData> {
+    const { data: riskData, pdfData } = await this.dilisenseService.getRiskData(name, dob);
+
+    // upload file
+    const { contentType, buffer } = Util.fromBase64(pdfData);
+    await this.documentService.uploadFile(
+      userData,
+      FileType.NAME_CHECK,
+      `${new Date().toISOString()}-nameCheck`,
+      buffer,
```

Comment:
Currently the file names look like this: `20250402-NameCheck-0-311124-163302.pdf`, I think we should keep the format more or less?

20250402 is the date
311124 is the user data ID
163302 is the time